app/boards/shields/nice_view/nice_view.zmk.yml

```diff
@@ -0,0 +1,8 @@
+file_format: "1"
+id: nice_view
+name: nice!view
+type: shield
+url: https://nicekeyboards.com/nice-view
+requires: [pro_micro, i2c_oled]
```

Comment:
We should consider this, this is an odd requirement, if we want this shield to be a bit more generic.